_posts/2018-06-13-hackillinois-2019-mission.markdown

```diff
@@ -0,0 +1,49 @@
+Something old, something new...
+
+
```

Comment:
I feel like this first paragraph could better set the tone for the discussion of the article below -- I know a lot of our previous articles have already discussed the event recap, OSS, and mentors, so we want this article to offer something different and clear right off the bat